### PR TITLE
Clarify schedule

### PR DESCRIPTION
**Pull request by iMylene**

### CODE REVIEW

**Review by mvr320 — CHANGES_REQUESTED** [review]
GHA fails.

File: content/schedule.md [hunk 1]
```diff
@@ -8,9 +8,10 @@ weight: 1
 
 The schedule of the day itself is still uncertain. The schedule of this year looks something like this:
 
-- Month 04, discussion with other associations on when the preliminaries can be held (organized by individual universities).
-- Month 06, possibly preliminaries
-- Month 09, the preliminaries
-- Month 10, BAPC21
+| Month     | Action                                                    |
+| --------- | --------------------------------------------------------- |
+| June      | Announcement preliminaries dates                          |
```
Comment (mvr320) [inline content/schedule.md:13]:
```suggestion
| June      | Announcement of preliminaries date                           |
```

**Comment by mvr320** [comment]
@iMylene can you squash this?

**Review by mvr320 — CHANGES_REQUESTED** [review]
Can you squash and make it 1 prelim date?

**Comment by EuiYeonJang** [comment]
> Can you squash and make it 1 prelim date?
I'm confused as to what this is referring?

**Comment by mvr320** [comment]
It was for Mylène to make this PR have 1 commit,


And instead of multiple dates we should have 1 proposal date, and hope that everyone can do that date.

On 13 May 2021, at 14:58, EuiYeonJang ***@***.******@***.***>> wrote:


Can you squash and make it 1 prelim date?
I'm confused as to what this is referring?

**Review by mvr320 — APPROVED** [review]

**Pull request merged**